### ISSUE TITLE
download bfin files!

### ISSUE THREAD

**Issue by lizzieinvancouver** [lizzieinvancouver/temporalvar#31]

**Comment by lizzieinvancouver** [comment]
@donahuem Can you check I am looking at the right Bfin files? Some are quite big and they seem to have data each year ... see BfinN_51995121-16.txt (12.6 MB). Were you thinking I should look elsewhere for these data? These files will be slow to download, load and merge into SummaryOut.

**Comment by donahuem** [comment]
I have updated the output of the new runs to include the ending Bfin on each row of SummaryOut.   The BfinOut files should have 1000yrs * 200 runs = 200000 lines of output.  I could round all the numbers to fewer digist to save memory...

**Comment by lizzieinvancouver** [comment]
@donahuem Okay! So once a run with these is ready we should be good to go. I will close this issue since it seems done. Thank you!
